events/tests/importers/test_kulke.py

```diff
@@ -236,9 +236,9 @@ def test__handle_removed_events(self):
             begin_date=now - timedelta(days=60),
         )
 
-        self.assertFalse(Event.objects.filter(id=event_1.id).exists())
-        self.assertTrue(Event.objects.filter(id=event_2.id).exists())
-        self.assertTrue(Event.objects.filter(id=event_3.id).exists())
+        self.assertFalse(Event.objects.filter(id=event_1.id, deleted=False).exists())
```

Comment:
This is a soft deleted event, right? If so would it be safer to test that:

```python
self.assertTrue(Event.objects.filter(id=event_1.id, deleted=True).exists())
```

Reply:
Yeah, very true.